### PR TITLE
Added validation to Drop csv upload. Improved error handling.

### PR DESCRIPTION
**Pull request by kellan-simiotics**
## How to test these changes?
I will attach some error samples.

### CODE REVIEW

**Comment by kellan-simiotics** [comment]
Attaching some error cases.

[cat_value_amount.csv](https://github.com/bugout-dev/engine/files/8723102/cat_value_amount.csv)
[dog_value_address.csv](https://github.com/bugout-dev/engine/files/8723104/dog_value_address.csv)
[non_address_value.csv](https://github.com/bugout-dev/engine/files/8723105/non_address_value.csv)

**Comment by peersky** [comment]
Could you add spinner on uploading CSV?

File: packages/moonstream-components/src/components/Dropper/Drop.tsx [hunk 1]
```diff
@@ -73,22 +74,66 @@ const DropCard = ({
...
+      try {
+        web3ctx.web3.utils.toChecksumAddress(value);
+      } catch (error) {
+        handleParsingError(
+          `Error parsing value: ${value}. Value in 'address' column must be a valid address.`
+        );
+      }
+    }
+    if (column == "amount") {
+      if (isNaN(parseInt(value))) {
```
Comment (peersky) [inline packages/moonstream-components/src/components/Dropper/Drop.tsx:114]: This statement fails if value is float

**Review by peersky — CHANGES_REQUESTED** [review]

**Review by peersky — APPROVED** [review]
lgtm

**Pull request merged**